codewars/8kyu/convertToString.js

```diff
@@ -0,0 +1,5 @@
+function booleanToString2(b){
+  }
+
+  booleanToString = be => `${be}`;
```

Comment:
unnecessary space at the beginning of the line

Reply:
Done